src/components/navigationItems/NavigationItems.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import BottomNavigationAction from '@mui/material/BottomNavigationAction'
+import { Link } from 'react-router-dom'
+import { NavigationProps } from '../../common/types/navigationTypes'
+import theme from '../../styles/createTheme'
+
+export const NavigationItems = ({
+  activePage,
+  navigationConfig,
+}: NavigationProps) => {
+  return navigationConfig.map(({ value, Icon, linkTo, size }) => (
+    <BottomNavigationAction
+      key={value}
+      value={value}
+      component={Link}
+      to={linkTo}
+      icon={
+        <Icon
+          color={activePage === value ? theme.palette.primary.main : '#AFB1B6'}
```

Comment:
I've added it to theme, but I couldn't add it to theme.palette, because it causes error.
As well as I want to remove some breakpoint values, I cannot do it.
(All the attempts were made after merging from WFW-XXX_type_theme)
<img width="931" alt="image" src="https://github.com/WeFriiends/WeFriiendsWeb/assets/324031/0de67dcc-a098-4e74-ba60-86147f4b7e02">

Reply:
Yeh, i checked TS of `palette`. From my perspective in this case makes sense to add smth like `custom Palette` and move there naveIcon grey

Reply:
@oce4ka `theme.customPalette.colorNavIcon` this way looks fine